fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -625,6 +626,31 @@ ACTOR Future<Void> commitBatch(
 	/////// Phase 1: Pre-resolution processing (CPU bound except waiting for a version # which is separately pipelined and *should* be available by now (unless empty commit); ordered; currently atomic but could yield)
 	TEST(self->latestLocalCommitBatchResolving.get() < localBatchNumber-1); // Queuing pre-resolution commit processing 
 	wait(self->latestLocalCommitBatchResolving.whenAtLeast(localBatchNumber-1));
+	double queuingDelay = g_network->timer() - timeStart;
+	if ((queuingDelay > (double)SERVER_KNOBS->MAX_READ_TRANSACTION_LIFE_VERSIONS / SERVER_KNOBS->VERSIONS_PER_SECOND ||
+	     (BUGGIFY && g_network->isSimulated() && deterministicRandom()->random01() < 0.01)) &&
```

Comment:
I think you can use `BUGGIFY_WITH_PROB` to control the frequency with which an activated buggify line gets triggered.